backend/wui/settings/testing.py

```diff
@@ -4,8 +4,49 @@
 
 DEBUG = True
 
-INSTALLED_APPS += [
-    "django_migration_linter",
+# Application definition
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    "authtools",
+    "rest_framework",
+    "django_admin_listfilter_dropdown",
+    "django_extensions",
+    "import_export",
+    "django_linear_migrations",
+    "common",
+    "library_sample_shared",
+    "library",
+    "sample",
+    "request",
+    "incoming_libraries",
+    "index_generator",
+    "library_preparation",
+    "pooling",
+    "flowcell",
+    "report",
+    "invoicing",
+    "usage",
+    "stats",
+    "metadata_exporter",
+    "drf_spectacular",
+    "corsheaders",
+]
+
+MIDDLEWARE = [
```

Comment:
I understand the concatenation (`+=` syntactic sugar appends to the list) had to be dropped because we want to add elements in a particular order. I can live with this repetition, but I have to wonder... on the unmodified dev.py, does it really work? We have MIDDLEWARE imported from base, and then we concatenate the rest of the middlewares... isn't then the order all wrong? Ultimately, do the corsheaders really work? ...and If so, are they really needed? (e.g. if we remove the corsheader middleware, does it stop working?)

Reply:
Please double-check with a dev deployment locally if vue is reachable. For example changing the make rule we fixed (switched from set-prod to set-testing)... does it work with set-dev? my guess is that it doesn't, and then we need to fix dev.py accordingly, to match what we have here... and without forgetting about the middlewares that are specific to dev (if any, e.g. DBT)

Reply:
Yes I am 100% certain that if we remove any of the imports or config of corsheaders from dev.py it won't work, I tried it two weeks back.